### PR TITLE
Form section

### PR DESCRIPTION
**Pull request by maury18**
# added the form section for contact me:
- CSS for the form.
- Form validation.
- Added the "Formspree".
- more description added for the "Readme" file
- cleared from linter errors
- added the dot(.) so the images can be seen it.
- added a box-shadow for the submit button once you pressed on it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -191,6 +191,32 @@ <h3 class="about-languajes--heading">Languajes</h3>
...
+            you need
+            built or a <br> project that needs coding. I’d <br> love to help with it </p>
+
+        <form class="container" action="https://formspree.io/f/mwkywowd" method="post">
+            <fieldset>
+                <input class="text" type="text" name="name" placeholder="Maury" maxlength="30" required>
+                <input class="email" type="email" name="email" placeholder="maury94@gmail.com" required>
+                <textarea class="text-area" name="message" cols="30" rows="10"
+                    placeholder="Write your message here" maxlength="500" required></textarea>
+                <button type="submit" class="submit">
```
Comment (youmari) [inline index.html:211]: - [ ] nice implementing button style effect on hover but you should also implement it when the button is pressed so please try to do so

File: index.html [hunk 2]
```diff
@@ -191,6 +191,32 @@ <h3 class="about-languajes--heading">Languajes</h3>
...
+                    Get it touch
+                </button>
+            </fieldset>
+        </form>
+        <img src="/img/form-last-rectangle.png" alt="" class="form-last-rectangle">
+        <img src="/img/form-last-circle.png" alt="" class="form-last-circle">
+        <img src="/img/last-black-rectangle.png" alt="" class="form-black-rectangle">
```
Comment (youmari) [inline index.html]: your images not working maybe they work on your machine but on all the machines to fix this please try to add a dot before the slash like this  `src="./img/form(the rest here)"`

**Review by youmari — CHANGES_REQUESTED** [review]
Hi, @maury18 @mikemtzp , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ NO linters error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 3]
```diff
@@ -386,3 +386,128 @@ li {
...
+  background-color: #6070ff;
+  color: #fff;
+  border: 0.2rem solid #fff;
+}
+
+.submit:active {
+  padding: 2.5rem 1.5rem;
+  -webkit-box-shadow: 0 0 1.3rem 1rem #fff;
+  box-shadow: 0 0 1.3rem 1rem #fff;
+}
```
Comment (BenMukebo) [inline css/styles.css:489]: - [OPTIONAL ] Nice Job implementing button style effect 👏🏻 👏🏻  however The Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Please use the some CSS properties from the Figma design 😄  ⬇️ 

buttongood

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi Team**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- The form is submitted successfully with valid input values. :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ OPTIONAL] This is a group project. Please let's include all the authors in the README file.
- [ OPTIONAL]  Please let's make sure that contribution to a group project is distributed equally among the members( commit messages should be fairly equal).

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 4]
```diff
@@ -386,3 +386,128 @@ li {
...
+.text-area {
+  padding: 2rem 1rem;
+  font-size: 1.7rem;
+  color: #172b4d;
+  font-weight: 400;
+  display: block;
+  width: 100%;
+  border-radius: 1rem;
+  margin-bottom: 1.5rem;
+  border: none;
```
Comment (BenMukebo) [inline css/styles.css:461]: - [OPTIONAL ] Kindly give 12px of padding to all placeholder or input value, as you can see on the Figma design. ⬇️  Remember, we have to sticks to the design [templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) in Figma as much as possible 😄 

Your design    |        Figma
---------:|:------------
figmabad | figmagood

**Pull request merged**